docs/iris/example_code/Meteorology/COP_1d_plot.py

```diff
@@ -86,7 +86,6 @@ def main():
     # Constrain the period 1860-1999 and extract the observed data from a1b
     constraint = iris.Constraint(time=lambda
                                  cell: 1860 <= cell.point.year <= 1999)
-    with iris.FUTURE.context(cell_datetime_objects=True):
```

Comment:
You have not unindented the code that used to be in the with block. This example will now no longer run.